src/harberger-lib/components/Info.re

```diff
@@ -75,182 +411,52 @@ let make = (~chain, ~tokenId: TokenId.t) => {
-         depositAvailableToWithdrawEth ++ " " ++ unit;
-       }
-       ->restr}
-      <br />
-      {showEthWithUsdConversion
-         ? <small>
-             {{
-                "(" ++ depositAvailableToWithdrawUsd ++ " USD)";
-              }
-              ->restr}
-           </small>
-         : React.null}
-    </p>
-    <p>
-      <small>
-        <strong>
-          {{
-             tokenName ++ "'s Patronage: ";
-           }
-           ->restr}
-          <Rimble.Tooltip
-            message={
-              "This is the total contribution that has been raised thanks to the wildcard, "
-              ++ tokenName
-            }
-            placement="top">
-            <span> {js|ⓘ|js}->restr </span>
-          </Rimble.Tooltip>
-        </strong>
-      </small>
-      <br />
-      {{
-         totalPatronage ++ " " ++ unit;
-       }
-       ->restr}
-      <br />
-      {showEthWithUsdConversion
-         ? <small>
-             {{
-                "(" ++ totalPatronageUsd ++ " USD)";
-              }
-              ->restr}
-           </small>
-         : React.null}
-    </p>
-    {switch (definiteTime) {
-     | Some(date) =>
-       <p>
-         <small>
-           <strong>
-             "Foreclosure date: "->restr
-             <Rimble.Tooltip
-               message={
-                 "This is the date the deposit will run out and the current owner will lose guardianship of "
-                 ++ tokenName
-               }
-               placement="top">
-               <span> {js|ⓘ|js}->restr </span>
-             </Rimble.Tooltip>
-           </strong>
-         </small>
-         <br />
-         {{
-            MomentRe.Moment.format("LLLL", date);
-          }
-          ->restr}
-         <br />
-         <small>
-           "( "->restr
-           <CountDown endDateMoment=date />
-           ")"->restr
-         </small>
-       </p>
-     | None => React.null
-     }}
-    {switch (daysHeld) {
-     | Some((daysHeldFloat, timeAquired)) =>
-       let timeAcquiredString = timeAquired->MomentRe.Moment.toISOString;
-       <p>
-         <small>
-           <strong>
-             "Days Held: "->restr
-             <Rimble.Tooltip
-               message={
-                 "This is the amount of time "
-                 ++ tokenName
-                 ++ " has been held. It was acquired on the "
-                 ++ timeAcquiredString
-                 ++ "."
-               }
-               placement="top">
-               <span> {js|ⓘ|js}->restr </span>
-             </Rimble.Tooltip>
-           </strong>
-         </small>
-         <br />
-         {daysHeldFloat->Js.Float.toFixed->restr}
-         <br />
-       </p>;
-     | None => React.null
-     }}
-  </React.Fragment>;
+  let currentPriceWei = QlHooks.usePrice(~chain, tokenId);
+  let priceString =
+    switch (currentPriceWei) {
+    | Foreclosed(price)
+    | Price(price) =>
+      price->Web3Utils.fromWeiBNToEthPrecision(~digits=4) ++ " USD"
```

Comment:
I think it's better to show 2 digits here, 4 makes it seem more than it is.

Reply:
Yes, totally agree. And it shouldn't be hardcoded to USD either...